### PR TITLE
Driver controls

### PR DESCRIPTION
**Pull request by c0d3rman**
Added driver controls.

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/Robot.java [hunk 1]
```diff
@@ -18,18 +16,15 @@
...
+		// Configure operator chooser
 		operatorChooser.addDefault(new DefaultOperator());
+		RobotMap.Component.navx.zeroYaw();
 	}
 
 	@Override
 	public void teleopInitialize() {
-		// teleopCommand = new Noop();
 		teleopCommand = new ChassisMove(RobotMap.Component.chassis, driverChooser.getSelected());
 		teleopCommand.start();
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/Robot.java:29]: Get rid of this?

File: src/org/usfirst/frc4904/robot/humaninterface/HumanInterfaceConfig.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 public class HumanInterfaceConfig {
 	public static final GearAlign gearAlign = new GearAlign();
+	public static final double XBOX_MINIMUM_THRESHOLD = 0.1;
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/humaninterface/HumanInterfaceConfig.java]: Should this be a deadzone for parity with JOYSTICK_DEADZONE?

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 3]
```diff
@@ -58,8 +58,12 @@ public void bindCommands() {
 		RobotMap.Component.driverXbox.dPad.right.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 90,
 			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
 		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
-		// Inverted airplane-style analog control
-		new Climb(() -> Math.max(0, -RobotMap.Component.driverXbox.rightStick.getY())).start();
+		RobotMap.Component.driverXbox.b.onlyWhileHeld(HumanInterfaceConfig.gearAlign);
+		RobotMap.Component.driverXbox.b.whenReleased(normalDrive);
+		// Inverted (airplane-style) analog gain control
+		new Climb(() -> Math.max(0,
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java:64]: This is getting a little big to be in a lambda question mark?

**Review by ajnadel — COMMENTED** [review]
@c0d3rman

File: src/org/usfirst/frc4904/robot/Robot.java (hunk 1 above)
Comment (c0d3rman) [reply src/org/usfirst/frc4904/robot/Robot.java:29]: No

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java (hunk 3 above)
Comment (c0d3rman) [reply src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java:64]: Na man

**Review by ajnadel — APPROVED** [review]

**Pull request merged**